src/onepirate/modules/withRoot.tsx

```diff
@@ -1,16 +1,42 @@
 import React from "react";
-import { ThemeProvider } from "@material-ui/core/styles";
-import CssBaseline from "@material-ui/core/CssBaseline";
+import { ThemeProvider, Theme, StyledEngineProvider } from "@mui/material/styles";
+import CssBaseline from "@mui/material/CssBaseline";
 import theme from "./theme";
 
+
+
+
+declare module '@mui/styles/defaultTheme' {
```

Comment:
What is this (and why are there three of them)?

Reply:
This is an error! I'm fixing it and I'll commit